.github/workflows/build-git-annex.yaml

```diff
@@ -77,18 +130,40 @@ jobs:
         shell: bash
         run: |
           if echo "${{ matrix.flavor }}" | grep -q "crippled" ; then
-            scripts/ci/setup_crippledfs /crippledfs 500
+            case `uname` in
+                Linux)
+                    scripts/ci/setup_crippledfs /crippledfs 500
+                    ;;
+                Darwin)
+                    scripts/ci/setup_crippledfs_macos crippledfs 500
+                    ;;
+                *)
```

Comment:
What about just handling OS specific logic within the script?

Reply:
if script sets `crippledfs` variable (may be make it `CRIPPLEDFS_PATH`) and `source`ed -- then next could be avoided completely.  the other variables of the script should be unset -- we do similar thing in https://github.com/datalad/datalad/blob/master/tools/ci/install-annex.sh

Reply:
Done.